nebula_ros/include/nebula_ros/continental/continental_ars548_hw_interface_wrapper.hpp

```diff
@@ -26,6 +26,7 @@
 #include <continental_srvs/srv/continental_ars548_set_vehicle_parameters.hpp>
 #include <geometry_msgs/msg/accel_with_covariance_stamped.hpp>
 #include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
+#include <std_msgs/msg/float32.hpp>
```

Comment:
Why is this needed here?

Reply:
Float32 is used in line 69. Maybe some other included header had previously been importing it, but due to some unrelated change, this became necessary here.

Reply:
Mmm did not get the answer, but the diff became outdated so I will mark this as solved